docs/proposal.md

```diff
@@ -2,7 +2,18 @@
 
 ## Motivation and purpose
 
-TBC
+- Why are creating this dashboard?
```

Comment:
Just nitpicking here, but could we change the hyphens to `###` headers in markdown?